src/components/Form/Form.js

```diff
@@ -0,0 +1,34 @@
+import useStore from '../../hooks/useStore';
+
+import StyledForm from './StyledForm';
+
+export default function Form() {
+	const addTask = useStore(state => state.addTask);
+
+	function handleSubmit(event) {
+		event.preventDefault();
+		const form = event.target;
+		const name = form.elements.task.value.trim();
```

Comment:
Why do you name the variable `name` and not `task`? In the end it is a name for a `task` right? :)